### DISCUSSION TITLE
Is possible to support delve?

### DISCUSSION THREAD

**Discussion by sword-jin** [mintoolkit/mint#12] [Ideas]
I plan to use mint in our project after the kubecon, but I found I can't debug a go program. I think we can integrate https://github.com/go-delve/delve to the go-debugger image?

**Comment by kcq** [comment]
@sword-jin Can you describe your setup? What did you do? What debug image did you use? Have you tried using the `lightruncom/koolkits:golang` debug image or did you use the default (aka busybox)?

**Reply by sword-jin** [reply]
This is my command:

```
mint --log-level debug debug --runtime k8s --namespace *** --pod ***-85cf6659f8-htj8m --debug-image lightruncom/koolkits:golang --target kinid-session --run-as-target-shell=false
```

By the way, our image is distroless image.

**Reply by kcq** [reply]
Delve should have been there in the golang koolkits image, but there's a bug there... Here's a quick fix alternative with delve reinstalled: `mintoolkit/koolkits-golang:latest` Note that the image is slightly dated and it'll be good to refresh it.

**Reply by sword-jin** [reply]
I see, the latest image is correct

**Reply by sword-jin** [reply]
The delve version is very old, this version has a lot of bugs.
![image](https://github.com/mintoolkit/mint/assets/11711621/aeaaef3d-d9df-4c00-b34b-e5171f4a9cb5)

**Reply by sword-jin** [reply]
This image is created two years ago, 

https://hub.docker.com/layers/lightruncom/koolkits/golang/images/sha256-0b4c629d9f9929b8067d3da8da40f8c565d5c66cb452ce0e1f4d03ea009f40cc
